src/AddDiveWish.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import Button from './Button.js';
+
+export default function AddListItem({ onAddDiveWish }) {
+  return (
+    <Form
+      aria-label="Add a dive destination to your wishlist"
+      onSubmit={handleAddDiveWish}
```

Comment:
I would probably rename the function to `handleSubmit` since this is the function that handles everything about the `submit` event. I mean it's not wrong to write it as you did, but if I am not mistaken it is common practice to name it `handleSubmit` 🙂 
```suggestion
      onSubmit={handleSubmit}
```

Reply:
Makes sense, thank you!